src/translations/locales/common_es.json

```diff
@@ -8,7 +8,7 @@
     "getDirections": "Obtener las direcciones",
     "makeABooking": "Hacer una reserva",
     "kmAway": "{{distance}}km de distancia",
-    "lastUpdated": "Last updated {{updatedAt}}",
+    "lastUpdated": "Última actualización {{updatedAt}}",
     "cancel": "Cancel"
```

Comment:
I'd use "cancelar" instead of "cancel" if possible

Reply:
Good catch! Cheers